sass.py

```diff
@@ -107,8 +107,16 @@ def from_lambda(cls, name, lambda_):
         :rtype: :class:`SassFunction`
 
         """
-        argspec = inspect.getargspec(lambda_)
-        if argspec.varargs or argspec.keywords or argspec.defaults:
+        if PY3:
+            argspec = inspect.getfullargspec(lambda_)
+            varargs, varkw, defaults = (argspec.varargs, argspec.varkw,
```

Comment:
in py35+ you'll also want to include `kwonlyargs`

Since this creates a disparate branch you'll probably want to include `# pragma: no cover` comments on each branch

Maybe something like this?

```python
argspec = getargspec(lambda_)
if PY2:  # pragma: no cover
    argspec = inspect.getargspec(lambda_)
    unsuported = (...)
    unsupported += (getattr(argspec, 'kwonlyargs', None),)
else:  # pragma: no cover
    argspec = inspect.getfullargspec(lambda_)
    unsupported_argspec_attrs = (argspec.varags, argspec.keywords, argspect.defaults)

if unsupported_argspec_attrs: ...
```

Note that I picked a subtly different form that uses `if PY2: else:` -- this makes the python3 branch continue to work in python4 (if it were ever to come out)